### ISSUE TITLE
Error when rendering <toast>: _c is not a function

### ISSUE THREAD

**Issue by Gaia-Nutrition** [ktsn/vuex-toast#5]
I imported vuex-toast using npm (version v0.1.0 was pulled)
the Toast component was imported into a single-file-component like this:

```
<template>
	<div class="container">
		<toast position="ne"></toast>
	</div>
</template>
<script>
	import { ADD_TOAST_MESSAGE } from 'vuex-toast'
	import { Toast } from 'vuex-toast'

	export default {
		name: 'ForgotPassword',
		data () {
			return {
				email: null,
				loading: false
			}
		},
		components: {
			Toast
		},
		...
	}
</script>
```

When I try to load this component, it throws `[Vue warn]: Error when rendering component <toast>: _c is not a function`

Any idea?

**Comment by ktsn** [comment]
What Vue/vue-loader version are you using?

**Comment by ktsn** [comment]
If you are using Vue v2.1.7 or later, I think it's already fixed on vuex-toast v0.1.1.

**Comment by Gaia-Nutrition** [comment]
@ktsn I was using Vue v2.0.1. Just updated vue, vue-loader, vue-template-compiler and vuex-toast to the latest stable and voilá 👍 
Thanks for the hint 😉
